### DISCUSSION TITLE
PR review commenting improvements

### DISCUSSION THREAD

**Discussion by chrisjrex** [community/community#57223] [Mobile]
When writing code blocks in pr comments, text is autocorrected and capitalized (if the first character on a line). Which is not correct for most code examples and a source of daily frustration.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.115.0 (100405004)
Device: iPhone15,2 (iOS 16.5)
</details>

**Comment by HenSquared** [comment]
Hey @chrisjrex , this is really good feedback. I can reproduce this on iOS. I'm going to create an issue to track this internally. Thanks!
